backend/src/tasks/test/cve.test.ts

```diff
@@ -73,6 +73,23 @@ describe('cve', () => {
         'vulnerability'
       );
     }
+
+    //check that the service key is populated in the Vulnerability model
+    const vulnerabilities2 = await Vulnerability.find({
```

Comment:
I don't think we need these extra snapshots here. Can we instead just add code in the loop above where we populate `services` and check if `vulnerability.service` is equal to `service`?